vignettes/imputation.Rmd

```diff
@@ -43,8 +43,9 @@ The simplest imputation rule is to set the missing parts to a fixed value. For
 example
 
```

Comment:
Another paragraph i think we should include here (and possibly in the functions documentation themselves) is:
In {admiral} we don't allow users to pick any single part of the date/time to impute, we only enable to impute up to a highest level, i.e. you couldn't choose to say impute months, but not days.

Reply:
Added (to the vignette and `derive_vars_dt()` and `derive_vars_dtm()`)